icons/hicolor/convert.sh

```diff
@@ -2,7 +2,7 @@
 for x in 16 22 32 48 64 256
 do
   mkdir -p ${x}x${x}/apps
-  for name in `ls scalable/apps/ | grep ".svg" | sed 's/.svg//g'`
+  for name in `cat new-icons.txt`
```

Comment:
Please also copy your explanation WHY there is such a list as a comment to this script.
(Or, if we use `grep -v '^#'` instead of `cat`, it can be a comment in the list file)